### DISCUSSION TITLE
Feature Requests - Variable Stroke Width, hide UI & Import .svg files

### DISCUSSION THREAD

**Discussion by ArtistBirdy** [keiwando/toonsquid#73] [Ideas]
1. We require a feature similar to After Effects' variable widths for selected vector strokes.

Example shown in this random video: 

https://user-images.githubusercontent.com/118920457/205017026-49c455d6-cabe-4e35-8d4a-bfd076b3ce51.mov

   ![4E9374E5-07F2-4E22-AF87-A6E6B27460A5](https://user-images.githubusercontent.com/118920457/203598439-551b6cb0-14f3-431c-b2a2-ee5a05829049.gif)

3. Also need a button/guesture to hide and unhide UI for drawing comfortably.

4. Importing ".svg" files, to animate them further.

**Comment by keiwando** [comment]
Thanks for the suggestions!
